planet/src/main/webapp/js/itinerary.js

```diff
@@ -7,6 +7,20 @@ function closeForm() {
     document.getElementById('add-event').style.display = 'none';
 }
 
+function handleHotelChange() {
+    if (typeof(Storage) !== "undefined") {
+        sessionStorage.setItem('hotel', document.getElementById('hotel-address').value);
+    } else {
+        alert ('Please update your browser'); 
```

Comment:
Is there an alternative solution for when one's browser doesn't work so the use can still use the itinerary builder?

Reply:
It seems like most browsers will support this except for internet explorer 8 or lower 
![image](https://user-images.githubusercontent.com/46226072/86795897-06a0e580-c03c-11ea-8f8a-bb4251068f76.png)
There are also two alternatives I can think of:
since session storage is just for remembering user's input, I can still allow users to see their itinerary while showing the alert box as a warning. Users will lose their input address every time they go on another page and come back, but the itinerary builder will work fine. 
Another solution is to store it in the database and I will have to figure out a way to distinguish it from the normal events both on the client side and server side. 
Could I address this problem later after I move things to firebase?